hmsclient/hmsclient.py

```diff
@@ -36,8 +36,10 @@ class HMSClient(ThriftHiveMetastore.Client):
     __client = None
     __isOpened = False
 
-    def __init__(self, iprot=None, oprot=None, host=None, port=None):
+    def __init__(self, iprot=None, oprot=None, host=None, port=None, user=None, groups=[]):
```

Comment:
I'd rather not have mutable structure as default arguments.

Can we also make it a `None` by default?